films to weather/common/Logics/FilmsSearch.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using films_to_weather.Apis.Entities;
+using films_to_weather.Apis.YandexWeather;
+using films_to_weather.Apis.YandexWeather.Entities;
+using films_to_weather.common;
+using films_to_weather.Common.Caching;
+using films_to_weather.Common.Entities;
+using films_to_weather.Translator;
+using Telegram.Bot;
+
+namespace films_to_weather.Common.Logics
+{
+    public class FilmsSearch
+    {
+        private static readonly IApiKinopoisk _api_Kinopoisk = new ApiKinopoisk();
+        private static readonly IApiWeather _apiWeather = new ApiWeather();
+        private static readonly IFilterCasheService _filterCasheService = new FilterCasheService();
+        private static readonly TelegramBotClient _botClient;
+        private static readonly Library _library;
+        private const string typeFilterGenres = "genres";
+
+        public async Task<FilmModel[]> GetFilm()
```

Comment:
```suggestion
        public async Task<FilmModel[]> GetFilms()
```
+ ByWeather можно докинуть